### PR TITLE
Feature ubication

### PR DESCRIPTION
**Pull request by paulotijero**

### CODE REVIEW

File: src/components/markerRestaurant.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/** @jsx jsx */
+import React from "react";
+import { Marker, Popup } from "react-leaflet";
+import L from "leaflet";
+import { jsx } from "@emotion/core";
+
+import { useSelectRestaurant } from "../action-hooks";
+import { usePosition } from "../selectors";
+
+const markerRest = new L.Icon({
```
Comment (stg101) [inline src/components/markerRestaurant.js]: What does L.Icon do ?

Comment (paulotijero) [reply src/components/markerRestaurant.js]: https://stackoverflow.com/questions/47723812/custom-marker-icon-with-react-leaflet

Comment (paulotijero) [reply src/components/markerRestaurant.js]: I use it to load a custom icon on the map https://stackoverflow.com/questions/47723812/custom-marker-icon-with-react-leaflet

Comment (stg101) [reply src/components/markerRestaurant.js]: Thanks!

**Pull request merged**